.gitignore

```diff
@@ -4,4 +4,5 @@ composer.phar
 vendor
 codeclimate.json
 coveralls.json
-clover.xml
\ No newline at end of file
+clover.xml
+/.project
```

Comment:
revert this, you can register your environment specific file into global .gitignore

Reply:
thanks for the review, just removed it manually :+1: